Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -214,6 +214,19 @@ theorem tendsto_pow_const_mul_const_pow_of_abs_lt_one (k : ℕ) {r : ℝ} (hr :
   rw [tendsto_zero_iff_norm_tendsto_zero]
   simpa [div_eq_mul_inv] using tendsto_pow_const_div_const_pow_of_one_lt k hr'
 
+/--For `k ≠ 0` and a constant `r` the function `r / n ^ k` tends to zero. -/
+lemma tendsto_const_div_pow (r : ℝ) (k : ℕ) (hk : k ≠ 0) :
+    Tendsto (fun n : ℕ => r / n ^ k) atTop (𝓝 0) := by
```

Comment:
This should be true in higher generality. See [Filter.Tendsto.div_atTop](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Order/Field.html#Filter.Tendsto.div_atTop)

Reply:
Ah yes that seems to be much more general. I've added a more general constant_div_atTop lemma and used it to prove the original one. I think its nice to have the specific one since its the one that shows up when doing L-series and modular form stuff. But if you think its not worth having as a lemma on its own I'm happy to remove it.

Reply:
If you really want the specialized version, then I suppose I'm okay with it, but notice that your proof is more work than necessary. This works:
```lean
  simpa using tendsto_const_div_atTop _ r <| tendsto_natCast_atTop_atTop.comp (tendsto_pow_atTop hk)
```